nf_core/pipeline-template/README.md

```diff
@@ -73,6 +73,19 @@ The results obtained from the full-sized test can be viewed on the [nf-core webs
 
 {% if branded -%}
 
+## Codespaces
+This repo includes a devcontainer configuration which will create a GitHub Codespace for Nextflow development!
+
+Devcontainer specs:
+- [DevContainer config](.devcontainer/devcontainer.json)
+- [Dockerfile](.devcontainer/Dockerfile)
+
+# Getting started
+- Open the repo in [Codespaces](../../codespaces)
```

Comment:
An absolute path is better as this will be rendered on the nf-core website too..

Reply:
Fixed